### PR TITLE
Delete duplicates

### PR DESCRIPTION
**Pull request by dshevtsov**
## Purpose of this pull request

This pull request (PR) deletes the duplicates identified using a regex `\s(\p{L}+) \1\s`.

### CODE REVIEW

**Review by dobooth — APPROVED** [review]
Nice regex.

**Comment by dshevtsov** [comment]
running tests

File: src/guides/v2.3/pattern-library/controls/buttons/buttons.md [hunk 1]
```diff
@@ -241,9 +241,9 @@ Left/Right: 15px
 
 #### Margin Spacing
 
-Top/Bottom margin spacing should should not be less than 10px
+Top/Bottom margin spacing should not be less than 10px
 
-Left/Right margin spacing should not be less than 10px
+Left/Right margin spacing not be less than 10px
```
Comment (jeff-matthews) [inline src/guides/v2.3/pattern-library/controls/buttons/buttons.md:246]: I this was deleted in error

Comment (dshevtsov) [reply src/guides/v2.3/pattern-library/controls/buttons/buttons.md:246]: Good catch

Comment (dshevtsov) [inline src/guides/v2.3/pattern-library/controls/buttons/buttons.md:246]:
```suggestion
Left/Right margin spacing should not be less than 10px
```

**Review by jeff-matthews — APPROVED** [review]

**Comment by dshevtsov** [comment]
running tests

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**